### PR TITLE
*: support metadata with multiple GTID sets

### LINKED ISSUE #678: do not support parse metedata with multiple GTID sets
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

start task for a MySQL with multiple GTID sets

```sql
> SHOW MASTER STATUS;
+------------------+----------+--------------+------------------+--------------------------------------------+
| File             | Position | Binlog_Do_DB | Binlog_Ignore_DB | Executed_Gtid_Set                          |
+------------------+----------+--------------+------------------+--------------------------------------------+
| mysql-bin.000003 | 1274     |              |                  | 5b5a8e4e-9b43-11ea-900d-0242ac170002:1-10, |
|                  |          |              |                  | 5b642cb6-9b43-11ea-8914-0242ac170003:1-7   |
+------------------+----------+--------------+------------------+--------------------------------------------+
```

2. What did you expect to see?

dump, load, then sync.

3. What did you see instead?

```
» query-status test-2
{
    "result": true,
    "msg": "",
    "sources": [
        {
            "result": true,
            "msg": "",
            "sourceStatus": {
                "source": "mysql-replica-01",
                "worker": "dm-worker-1",
                "result": null,
                "relayStatus": null
            },
            "subTaskStatus": [
                {
                    "name": "test-2",
                    "stage": "Paused",
                    "unit": "Sync",
                    "result": {
                        "isCanceled": false,
                        "errors": [
                            {
                                "msg": "parse GTID 5b5a8e4e-9b43-11ea-900d-0242ac170002:1-10,: invalid GTID format, must UUID:interval[:interval]",
                                "error": null
                            }
                        ],
                        "detail": null
                    },
                    "unresolvedDDLLockID": "",
                    "sync": {
                        "totalEvents": "0",
                        "totalTps": "0",
                        "recentTps": "0",
                        "masterBinlog": "(mysql-bin.000003, 1274)",
                        "masterBinlogGtid": "5b642cb6-9b43-11ea-8914-0242ac170003:1-7,5b5a8e4e-9b43-11ea-900d-0242ac170002:1-10",
                        "syncerBinlog": "(, 4)",
                        "syncerBinlogGtid": "",
                        "blockingDDLs": [
                        ],
                        "unresolvedGroups": [
                        ],
                        "synced": false,
                        "binlogType": "remote"
                    }
                }
            ]
        }
    ]
}
```

4. Versions of the cluster

    - DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):

        ```
        master branch before git commit hash 83063141c5fd29058377336e90e448027be6206d
        ```

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

fix #678 

### What is changed and how it works?

combine multiple GTID sets cross multiple lines into one string without `\n`.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@WangXiangUSTC @lichunzhu PTAL

File: pkg/utils/mydumper.go [hunk 1]
```diff
@@ -81,3 +87,33 @@ func ParseMetaData(filename string) (mysql.Position, string, error) {
...
+	var following strings.Builder
+	for {
+		line, err := br.ReadString('\n')
+		if err == io.EOF {
+			return following.String(), nil // return the previous, not including the last line.
+		} else if err != nil {
+			return "", err
+		}
+
+		line = strings.TrimSpace(line[:len(line)-1])
```
Comment (WangXiangUSTC) [inline pkg/utils/mydumper.go:101]: use `line[:len(line)-1]` to trim `\n`? seems that `TrimSpace` can trim `\n`

Comment (csuzhangxc) [reply pkg/utils/mydumper.go]: you are right. addressed in https://github.com/pingcap/dm/pull/679/commits/fd00838494d25dd5cbe8d4acd76995fbf54c3c90.

**Pull request merged**